packit_service/worker/helpers/testing_farm.py

```diff
@@ -1380,6 +1387,12 @@ def report_status_to_configured_job(
         links_to_external_services: Optional[Dict[str, str]] = None,
         update_feedback_time: Callable = None,
     ):
+        if description == TASK_ACCEPTED and self.job_config.manual_trigger:
```

Comment:
will this work in case the test is triggered manually (in that case we want to show that status)?

Reply:
No, this is always hidden. I thought about trying to show it when you really triggered the test manually. But I think this will require some code (as putting this information into the database) and I don't know if it is worth it.

I would prefer to work on adding dependencies to our code instead of trying to improve this workaround. So that we could show all the messages that now I am hiding at the proper moment.